src/OrchardCore.Modules/OrchardCore.Cors/Assets.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "inputs": [
+      "Assets/Admin/cors.js"
```

Comment:
If you call it cors-admin in output, you could also call it cors-admin.js instead of Admin/cors/js.